nucypher-core-wasm/Cargo.toml

```diff
@@ -20,14 +20,15 @@ default = ["console_error_panic_hook"]
 
 [dependencies]
 # TODO: Replace with a new release after this PR is merged and released: https://github.com/nucypher/rust-umbral/pull/122
-umbral-pre = { git = "https://github.com/piotr-roslaniec/rust-umbral.git", rev = "90a5075bb3462421fa6db79463960cd8e10a7025", features = ["bindings-wasm"] }
-ferveo = { package = "ferveo-pre-release", version = "0.1.0-alpha.7", features = ["bindings-wasm"] }
+umbral-pre = { git = "https://github.com/piotr-roslaniec/rust-umbral.git", rev = "5a279039d1e44e415f2cfac85282fc5102d41a6c", features = ["bindings-wasm"] }
+ferveo = { package = "ferveo-pre-release", git = "https://github.com/nucypher/ferveo.git", rev = "9795e2969f525fc7ee3a7721dca368287f4825c8", features = ["bindings-wasm"] }
 nucypher-core = { path = "../nucypher-core" }
 wasm-bindgen = "0.2.86"
 js-sys = "0.3.63"
 console_error_panic_hook = { version = "0.1", optional = true }
 derive_more = { version = "0.99", default-features = false, features = ["from", "as_ref"] }
-wasm-bindgen-derive = "0.2.0"
+# TODO: Replace with a new release after this issue is fixed: https://github.com/fjarri/wasm-bindgen-derive/issues/3
+wasm-bindgen-derive = { git = "https://github.com/piotr-roslaniec/wasm-bindgen-derive.git", rev = "d13475a9b36d6c6e5f2f99372d175f21b00bf0d6" }
```

Comment:
Does the `[profile.release]` in `wasm-bindgen-derive` really affect the build of `nucypher-core`? https://doc.rust-lang.org/cargo/reference/profiles.html states "Profile settings defined in dependencies will be ignored."

Reply:
This can be updated now to v0.2.1.